src/ui/widgets/calendar.rs

```diff
@@ -0,0 +1,111 @@
+use super::*;
+
+const DAY_SIZE: (u16, u16) = (4, 3);
```

Comment:
what is 4 and 3? magic?
this is:
const DAY_WIDTH: u16 = 4;
const DAY_HEIGHT: u16 = 3;

I guess

Reply:
I think the name DAY_SIZE is self explanatory, but if you want I can change it